Libraries/Components/Touchable/TVTouchable.js

```diff
@@ -36,7 +36,11 @@ export default class TVTouchable {
     this._tvEventHandler = new TVEventHandler();
     this._tvEventHandler.enable(component, (_, tvData) => {
       tvData.dispatchConfig = {};
-      if (ReactNative.findNodeHandle(component) === tvData.tag) {
+      let tag;
+      try {
+        tag = ReactNative.findNodeHandle(component);
+      } catch (e) {}
```

Comment:
I am not sure that this is the right fix for the issue.

The error that is thrown by `findNodeHandle()` in dev mode is important to know about, so we don't want to always suppress it.

Seems like there might be a deeper issue -- maybe unmounting is not cleaning up or disabling the handler correctly or in the right order.

Reply:
Does this problem occur with Fabric turned on, or with Fabric turned off?

Reply:
It's fabric turned off.